frontend/src/metabase/dashboard/components/grid/utils.js

```diff
@@ -17,6 +17,16 @@ export function sortCardsForMobile(a, b) {
   return a.x - b.x;
 }
 
+export const MOBILE_HEIGHT_BY_DISPLAY_TYPE = {
```

Comment:
It would be nice to keep the size apsect of cards in a single place. Currenty, the cards sizing is defined in `frontend/src/metabase/visualizations/shared/utils/sizes.ts`

Reply:
Good call. I'll move the size related const values 👍